lib/grape/railtie.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+module Grape
+  class Railtie < Rails::Railtie
+    if Rails.version.to_f >= 6.0
+      initializer 'grape, setup zeitwerk loader for custom validators' do |_app|
+        Grape.config.on_unknown_validator << method(:on_grape_unknown_validator) if Rails.autoloaders.zeitwerk_enabled?
+      end
+    end
+
+    def on_grape_unknown_validator(name)
```

Comment:
If the config option is `validator`, this also can just be `validator` since it exists in the `Grape::Railtie` namespace.